extension/app/src/lib/utils.ts

```diff
@@ -0,0 +1,62 @@
+export const randomString = (length: number): string => {
```

Comment:
In Auth0 [doc](https://auth0.com/docs/get-started/authentication-and-authorization-flow/authorization-code-flow-with-pkce/call-your-api-using-the-authorization-code-flow-with-pkce#create-code-verifier) they explain you need to send a base64, here you are using a base66.

Reply:
Fixed, not using this anymore!